content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure)
+files, uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to dvc via `dvc add` or
+`dvc commit` or `dvc repro` but the metafiles are not committed to git yet. The
+second change shows the new changes to the data that is not tracked by DVC yet
+or what you could `dvc commit`. The third change shows the files that are
+missing from the cache. Some of those could be pulled from the remote.
+
+Using `--untracked-files`, it'll show the files that are not being tracked by
+DVC or Git.
+
+By default, `dvc data status` shows changes in the level the file/directory is
+being tracked in. With `--granular`, it will show changes in file-level.
+
+The `--unchanged` option will list all of the files that aren't listed in
+committed changes or uncommitted changes.
+
+### Output
+
+The `dvc data status` displays changes in multiple categories such as:
+
+- _Not in cache_
+- _DVC committed changes_
+- _DVC uncommitted changes_
+- _Untracked files_
+- _DVC unchanged files_
+
+The categories will only be displayed when there are changes related to those
+categories.
+
+The _DVC committed changes_ and _DVC committed changes_ shows more detailed
+state the files are in: _added_, _modified_, and _deleted_.
+
+## Options
+
+- `--granular` - Show granular, file-level information for the changes for
+  dvc-tracked directories. By default, DVC only shows the changes at the level
+  where the data is being tracked at.
+
+- `--untracked-files` - Show files that are not being tracked by DVC and Git.
+  This option can also take values: `no` and `all`. With `--untracked-files=no`,
+  this change is not shown and is a default. `--untracked-files=all` or just
+  `--untracked-files` will show untracked files.
+
+- `--unchanged` - Show unchanged DVC-tracked files.
+
+- `--json` - prints the command's output in easily parsable JSON format, instead
+  of a human-readable output.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to update your local storage)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
```

Comment:
Should we update the message here to suggest `dvc destroy` to stop tracking the file?

Reply:
Edit: I meant `dvc remove`.

Reply:
`dvc remove` might be more suitable in `DVC committed changes`, no?  It untracks the output instead of rolling back.

Reply:
I was thinking if you do `rm model.pkl`, it will show as an uncommitted change in `deleted` status. You may want to also do `dvc remove model.pkl.dvc` to stop tracking the file.

Anyway, I think it's a bit hard to guess what to suggest here, and it's probably better left as is for now.